lib/validators/resolveNestedDefinitions.ts

```diff
@@ -76,7 +76,9 @@ export function resolveNestedDefinitions(spec: SwaggerObject): SwaggerObject {
   // a function to resolve SchemaObject (top-level and nested)
   const resolveSchemaObject = (schemaObject: SchemaObject): SchemaObject => {
     const result = propertySetMap<SchemaObject>(schemaObject, {
-      properties: properties => stringMapMap(properties, resolveNestedSchemaObject),
+      properties: skipIfUndefined(properties =>
```

Comment:
Can you help me understand if this change caused by the change above of allowNullType set for top level? I don't see the relevance when try locally.

Reply:
Per my understanding, this part is not quite related to above change